espanol/bailar/matar/matar.md

```diff
@@ -0,0 +1,4 @@
+
+Beatito sacó un arma del bolsillo y mató a quemarropas al consejero.
+
+Luego, se preparó para recibir a todas las milicias, con quienes ya había acordado el traspaso del poder, y la construcción de una cómoda mansión para él y su joven esposa con vista a toda el desierto del Sertao.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.